internal/soak_tests/map/main.go

```diff
@@ -212,7 +212,7 @@ func main() {
 			}
 		}
 	}(ctx, sts)
-	c := make(chan os.Signal)
+	c := make(chan os.Signal, 1)
```

Comment:
What is the difference between the two?

Reply:
Channels carrying os.Signal is recommended to be buffered.